runtime/sam/expr/cast/overflow.go

```diff
@@ -0,0 +1,141 @@
+package cast
```

Comment:
Nit: No need to export this stuff.

Reply:
For a follow up pr it needs to be exported.